Sources/Amplitude/Amplitude.m

```diff
@@ -880,22 +871,44 @@ - (NSDictionary *)mergeEventsAndIdentifys:(NSMutableArray *)events identifys:(NS
                     ([[event objectForKey:SEQUENCE_NUMBER] longLongValue] <
                      [[identify objectForKey:SEQUENCE_NUMBER] longLongValue])) {
                 [events removeObjectAtIndex:0];
-                maxEventId = [[event objectForKey:EVENT_ID] longValue];
                 identify = nil;
             } else {
                 [identifys removeObjectAtIndex:0];
-                maxIdentifyId = [[identify objectForKey:EVENT_ID] longValue];
                 event = nil;
             }
         }
 
         [mergedEvents addObject:event != nil ? event : identify];
     }
 
-    NSDictionary *results = [[NSDictionary alloc] initWithObjectsAndKeys:mergedEvents, EVENTS, [NSNumber numberWithLongLong:maxEventId], MAX_EVENT_ID, [NSNumber numberWithLongLong:maxIdentifyId], MAX_IDENTIFY_ID, nil];
+    NSDictionary *results = [[NSDictionary alloc] initWithObjectsAndKeys:mergedEvents, EVENTS, nil];
     return results;
 }
 
+- (NSMutableArray *)removeEventFromBuffer:(NSArray *)buffer currentEventString:(NSString *)currentEventString {
+    NSMutableArray *updatedBuffer = [buffer mutableCopy];
+    NSUInteger index = 0;
+    for (NSDictionary *event in buffer) {
+        NSMutableArray *currentEventDitconary = [[NSMutableArray alloc] init];
```

Comment:
This is because the currentEventString with the form like `[{xxx}]`, I want to make the format of currentEventDitconary the same as the currentEventString. An alternative way is to remove the first and last character in the currentEventString, Feels like it's more efficient than new a NSMutableArray every time.